elements/pf-popover/pf-popover.ts

```diff
@@ -391,27 +398,26 @@ export class PfPopover extends LitElement {
     super.disconnectedCallback();
     PfPopover.instances.delete(this);
     this.#referenceTrigger?.removeEventListener('click', this.toggle);
-    this.#referenceTrigger?.removeEventListener('keydown', this.onKeydown);
+    this.#referenceTrigger?.removeEventListener('keydown', this.#onKeydown);
   }
 
   #getReferenceTrigger() {
     const root = this.getRootNode() as Document | ShadowRoot;
     return !this.trigger ? null : root.getElementById(this.trigger);
   }
 
-
   #triggerChanged() {
     const oldReferenceTrigger = this.#referenceTrigger;
     this.#referenceTrigger = this.#getReferenceTrigger();
     if (oldReferenceTrigger !== this.#referenceTrigger) {
       oldReferenceTrigger?.removeEventListener('click', this.toggle);
-      oldReferenceTrigger?.removeEventListener('keydown', this.onKeydown);
+      oldReferenceTrigger?.removeEventListener('keydown', this.#onKeydown);
       this.#referenceTrigger?.addEventListener('click', this.toggle);
-      this.#referenceTrigger?.addEventListener('keydown', this.onKeydown);
+      this.#referenceTrigger?.addEventListener('keydown', this.#onKeydown);
     }
   }
 
-  @bound private onKeydown(event: KeyboardEvent) {
+  @bound private #onKeydown(event: KeyboardEvent) {
```

Comment:
This change causes `[ERROR] Expected identifier but found "#onKeydown"` given the @bound decorator should still use the `private _onKeydown()` format as the `#` still doesn't work with the decorators.

Reply:
this was resolved in source